src/Engine/Party.h

```diff
@@ -286,21 +286,38 @@ struct Party {
     /**
      * Return id of random character that can still act.
      *
-     * @return        ID of character or -1 if none of the character cat act.
+     * @param isGameRng   Use 'grng' if true, 'vrng' otherwise.
+     * @return            ID of character or -1 if none of the character cat act.
      */
-    int getRandomActiveCharacterId() const {
+    int getRandomActiveCharacterId(bool isGameRng) const {
```

Comment:
Let's just pass in grng / vrng explicitly at the call site. Thus:
```
getRandomActiveCharacterId(grng);
```
instead of
```
getRandomActiveCharacterId(true);
```

This way it would be way easier to parse what's going on right at the call site.